status/barcode.py

```diff
@@ -23,45 +23,89 @@ def get(self):
         self.write(t.generate(gs_globals=self.application.gs_globals, user=self.get_current_user()))
 
     def post(self):
-        # how often is the label(s) to be printed? Default: 1
-        copies = int(self.get_argument('copies'))
+        # some variable will only be attempted to be read if a specific form is submitted
+        labelType = self.get_argument("formType")
```

Comment:
We are trying to stick to single quotes for string in python now, for consistency's sake, not a big issue though.